CDP4ReferenceDataMapper.Tests/packages.config

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="MSTest.TestAdapter" version="2.1.1" targetFramework="net452" />

```

Comment:
i am surprised this file is here, should these references not be in the .csproj file?